packages/engine/Source/Scene/Scene.js

```diff
@@ -1871,31 +1873,36 @@ const requestRenderModeDeferCheckPassState = new Cesium3DTilePassState({
 
 const scratchOccluderBoundingSphere = new BoundingSphere();
 let scratchOccluder;
-
+/**
+ * Get the central body occluder for the scene.
+ *
+ * TODO: The occluder is the top-level globe. When we add
```

Comment:
Similar comment about TODO comments in the code. Would this be more helpful if it lived in an issue instead?

Reply:
In this case, I couldn't find any discussion about "support for multiple central bodies," so I don't think an issue would be appropriate.
I changed the comment to simply describe the assumptions made by this method, so that if someone _does_ want to discuss multiple bodies, they will understand what to change.